toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ja_JP">
+<context>
+    <name>ArrowToolOptionsBox</name>
+    <message>
+        <source>Pick:</source>
+        <translation>選択オブジェクト：</translation>
```

Comment:
No, just overlooked this part. Since the files are so long and the lines similar it is easy to get lost.